src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -907,8 +907,8 @@ public static class ParameterIdentification
          public static readonly string RemoveLLOQMode = "Remove data below LLOQ";
          public static readonly string TimeProfileAnalysis = "Time Profile";
          public static readonly string PredictedVsObservedAnalysis = "Predicted vs. Observed";
-         public static readonly string SimulatedChartAxis = "Simulated ";
-         public static readonly string ObservedChartAxis = "Observed ";
+         public static readonly string SimulatedChartAxis = "Simulated";
```

Comment:
It's not easy to use those constants when there is " " at the end. Instead, let's add the space where it's being used